build.sh

```diff
@@ -11,12 +11,12 @@ function copySrcToBuild {
   rm -rf build/
   mkdir build/
 
-  cp -r src/ build/
+  cp -r src/. build/
```

Comment:
It indicates that I want to copy the contents of src, not src itself. I assume that must have worked for the build job anyway, but locally it copied the src folder, so I had build/src, which is not what the build.sh expects.